src/color_tft.toit

```diff
@@ -116,6 +82,48 @@ COLOR_TFT_16_PIXEL_MODE_ ::= 5
 COLOR_TFT_18_PIXEL_MODE_ ::= 6
 COLOR_TFT_TRUNCATED_24_PIXEL_MODE_ ::= 7
 
+/**
+TrueColor Driver intended to be used with the Pixel-Display package
+  at https://pkg.toit.io/package/pixel_display&url=github.com%2Ftoitware%2Ftoit-pixel-display&index=latest
+See https://docs.toit.io/language/sdk/display
```

Comment:
I'm hoping the regexps for finding URLs are smart enough to remove trailing dots.
Up to you. (If you think it wouldn't work, leave them out.)
```suggestion
  at https://pkg.toit.io/package/pixel_display&url=github.com%2Ftoitware%2Ftoit-pixel-display&index=latest.
See https://docs.toit.io/language/sdk/display.
```